TR-010 Incremento

RickGutler opened the pull request:
Criado o update de produtor.
Feito o DTO da atualização de dados.
Criado o método na classe Produtor para atualização correta.

RickGutler committed:
TR-010 Incremento

Criado o update de produtor.
Feito o DTO da atualização de dados.
Criado o método na classe Produtor para atualização correta.

Prezotti reviewed (changes requested):
Criar ProdutoService com o método atualizarProduto() e o controller passará a chamar o service e o service chama o repository

RickGutler committed:
TR-010 Incremento

Feito o Atualizar de Produtor.
Criado o Service de Produtor para verificar a existência do produtor ao atualizar.

Prezotti reviewed (changes requested):
A função atualizarProdutor no service não pode ser void, ela tem que retornar um produtor atualizado.
No controller seu produtor vai receber service.atualizarProdutor, e ele não vai ser mais o getReferenceById porque você ainda não testou se o id existe no controller. Voce so testa isso no service

RickGutler committed:
TR-010 Incremento

Feito o Atualizar de Produtor.
Criado o Service de Produtor para verificar a existência do produtor ao atualizar.

Prezotti reviewed (approved):
TOP!
